tests/installation/partitioning_raid.pm

```diff
@@ -158,6 +164,97 @@ sub set_lvm {
     wait_screen_change { send_key $cmd{finish} };
 }
 
+sub modify_uefi_boot_partition {
+    assert_screen 'partitioning_raid-disk_vdd_with_partitions-selected';
+    # fold the drive tree
+    send_key 'left';
+    assert_screen 'partitioning_raid-hard_disks-unfolded';
+    # select first disk
+    send_key 'right';
+    assert_screen 'partitioning_raid-disk_vda_with_partitions-selected';
+    # edit first partition
+    send_key 'alt-e';
+    assert_screen 'partition-format';
+    # format as FAT (first choice)
+    send_key 'alt-a';
+    assert_screen 'partitioning_raid-format_fat_UEFI';
+    # mount point selection
+    send_key 'alt-o';
+    assert_screen 'partitioning_raid-mount_point-focused';
+    # enter mount point
+    type_string '/boot/efi';
+    assert_screen 'partitioning_raid-mount_point_boot_efi';
+    send_key $cmd{finish};
+    assert_screen 'expert-partitioner';
+    send_key 'shift-tab';
+    send_key 'shift-tab';
+    # go to top "Hard Disks" node
+    send_key 'left';
+    assert_screen 'partitioning_raid-hard_disks-unfolded';
+    # fold the drive tree again
+    send_key 'left';
+}
+
+sub add_bios_boot_partition {
+    record_soft_failure 'bsc#1063844';    # Cannot add partition from menu option "hard disks"
+    send_key_until_needlematch 'partitioning_raid-disk_vda-selected', 'down';
+    addpart 'bios-boot';
+    send_key 'down';
+    send_key_until_needlematch 'custompart', 'left';
+    send_key 'alt-s';                     #System view
+    send_key_until_needlematch 'partitioning_raid-hard_disks-unfolded', 'right';
+}
+
+sub add_prep_boot_partition {
+    if (is_storage_ng) {
+        record_soft_failure 'bsc#1063844';    # Cannot add partition from menu option "hard disks"
+        send_key 'down';
+        assert_screen 'partitioning_raid-disk_vda-selected';
+        send_key 'alt-d';
+    }
+    else {
+        send_key 'alt-p';
+    }
+    if (!get_var('UEFI')) {                   # partitioning type does not appear when GPT disk used, GPT is default for UEFI
+        if (is_storage_ng) {
+            record_soft_failure 'bsc#1055743';    # No partitioning type page ATM
```

Comment:
check the bug - currently it has state "VERIFIED INVALID"

Reply:
You're right, we can fix that here as well.

Reply:
Have changed that in 2 places